src/feature/stats/bw_array_st.h

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
I think in Tor the general purpose of files ending `_st.h` is to contain *only* the `struct` and no methods related to the struct.